.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/wjakob/nanobind
 [submodule "cmake-module"]
 	path = cmake-module
-	url = https://github.com/jrl-umi3218/jrl-cmakemodules.git
+	url = https://github.com/nim65s/jrl-cmakemodules.git
```

Comment:
This change should be reverted before merging the branch.
We should wait for the following PR https://github.com/jrl-umi3218/jrl-cmakemodules/pull/776